src/differential_arithmetic.jl

```diff
@@ -67,11 +67,15 @@ for T in (:One, :AbstractThunk, :Any)
 end
 
 
-Base.:+(a::One, b::One) = extern(a) + extern(b)
+strong_one(primal::Number) = true
```

Comment:
It has been pointed out by @simeonschaub  that if we do 
`strong_one(primal::Number) = I` 
this work out well.
Which solves a bunch of problems of not knowing what the right `strong(1)` is, and kinda removes the need for a `strong_one` function in the first place?

Reply:
No, not really. Strong one probably shouldn't change the number type.
```julia
julia> I * 1.5
UniformScaling{Float64}
1.5*I
```